testfiles/gedcom70/escapes.ged

```diff
@@ -6,4 +6,24 @@
 1 NOTE me@example.com is an example email address.
 2 CONT @@me and @I are example social media handles.
 2 CONT @@@@@ has four @ characters where only the first is escaped.
+0 @N01@ SNOTE @@ one leading
+0 @N02@ SNOTE @@one leading no space
+0 @N03@ SNOTE @@ two leading
+0 @N04@ SNOTE @@two leading no space
+0 @N05@ SNOTE doubled @ internal
+0 @N06@ SNOTE doubled@internal no space
```

Comment:
These duplicate the next lines, don't match their own text

Reply:
Updated